swan-lake/development-tutorials/deployment-guide/serverless-deployment.md

```
@@ -83,18 +133,87 @@ The AWS Lambda functionality is implemented as a compiler extension. Therefore,
+
+>**Info:** The functions are deployed on the `provided.al2023` custom runtime. The `provided` runtime, which earlier versions used, has been retired by AWS and can no longer be used to create or update a function.
 
 >**Info:** For the supported parameters, go to the <a href="https://docs.aws.amazon.com/cli/latest/reference/lambda/create-function.html" target="_blank">`create-function` documentation</a>. You might need to change parameters such as the `MemorySize` and `Timeout` depending on your application and connection speed.
 
+### Redeploy the function
+
+After the function is created, you can push a new build of it by executing the `aws lambda update-function-code` command, which you also get in the CLI output logs after you [build the function](#build-the-function). The command uploads the regenerated ZIP file, or refers to the new image with the `--image-uri` parameter when the function is [deployed as a container image](#deploy-the-function-as-a-container-image). The runtime, the role, and the layer are configured when the function is created. Therefore, they are not repeated when you redeploy.
+
+A function created before AWS retired the `provided` runtime in February 2024 is still configured with a runtime that can no longer be selected, and it cannot be redeployed as it is. Move it to `provided.al2023` once by executing the command below, and then redeploy as usual.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)